### PR TITLE
add loopring to rollup_economics, corrected imx + starkware proofs

### PR DESCRIPTION
**Pull request by lorenz234**
- added loopring to the models/rollup_economics/ethereum/l1_data_fees.sql table
- fixed imx, as it uses offchain DA. therefore moved all state update fees into l1_data_fees
- added starkware proofs to the models/rollup_economics/ethereum/l1_verification_fees.sql table
- double checked all proof sizes

### CODE REVIEW

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Comment by jeff-dude** [comment]
i've also pushed a commit to update the spell(s) to filter on date ranges as expected, to see if that helps run time

**Comment by jeff-dude** [comment]
> i've also pushed a commit to update the spell(s) to filter on date ranges as expected, to see if that helps run time

it's possible the previous CI test run was at the same time as other PRs, therefore less resources, but it's now running in ~17 minutes compared to almost an hour previously. this looks much better. i'd still suggest the above change, if you don't mind, prior to merge

File: models/rollup_economics/ethereum/l1_data_fees.sql [hunk 1]
```diff
@@ -142,46 +157,55 @@ with tx_batch_appends as (
...
+    ON p.minute = date_trunc('minute', t.block_time)
+    AND p.blockchain is null
+    AND p.symbol = 'ETH'
+    {% if is_incremental() %}
+    AND p.minute >= date_trunc('day', now() - interval '7' day)
+    {% endif %}
+  WHERE (
+      t."from" = 0x2c169dfe5fbba12957bdd0ba47d9cedbfe260ca7 -- StateUpdate poster
+      AND t.to = 0xc662c410C0ECf747543f5bA90660f6ABeBD9C8c4 -- StateUpdate proxy contract
+      AND cast(t.data as varchar) LIKE '0x77552641%'
```
[the remark this answers is not in the document]
Comment (lorenz234) [reply models/rollup_economics/ethereum/l1_data_fees.sql:171]: Sure thing, I can do it!

**Comment by lorenz234** [comment]
Now 18min lol. But did some query testing, and it does seem to reduce cpu time by around 50%. Should be ready for merge now!

**Comment by jeff-dude** [comment]
> Now 18min lol. But did some query testing, and it does seem to reduce cpu time by around 50%. Should be ready for merge now!

we've likely hit the best we can get on this side, which is fine! thanks again for working with me on enhacements here.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**